sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Covariance.scala

```diff
@@ -76,8 +76,14 @@ abstract class Covariance(x: Expression, y: Expression) extends DeclarativeAggre
   }
 }
 
+
```

Comment:
This wouldn't be a nit because most of case class definitions across expressions seem have double-spaced indentation. Also, it seems fine - https://github.com/databricks/scala-style-guide#blank-lines-vertical-whitespace

> Use one or two blank line(s) to separate class definitions.